apps/ledger-live-mobile/src/newArch/features/Accounts/components/AccountListView/components/AccountRow.tsx

```diff
@@ -0,0 +1,68 @@
+import React from "react";
+import { Flex, Tag, Text } from "@ledgerhq/native-ui";
+import ParentCurrencyIcon from "~/components/ParentCurrencyIcon";
+import BigNumber from "bignumber.js";
+import CounterValue from "~/components/CounterValue";
+import { Account, DerivationMode } from "@ledgerhq/types-live";
+import { useMaybeAccountName } from "~/reducers/wallet";
+import { getTagDerivationMode } from "@ledgerhq/coin-framework/lib/derivation";
+
+interface AccountRowProps {
+  account: Account;
+  balance: BigNumber;
+}
+
+const formatAddress = (address: string) => {
```

Comment:
extract this function to a `helpers.ts` or `utils.ts` script

Reply:
add a unit test on it please :)

Reply:
you can add it under `features/Accounts` so that all utility functions will be placed there

Reply:
Added in utils